src/components/smartLink.tsx

```diff
@@ -1,3 +1,6 @@
+'use client';
+
+import {useCallback} from 'react';
```

Comment:
I am not very familiar with NextJS, but maybe if we don't use `useCallback` we could still have server side rendering of this component?

Reply:
No the click handler is the problem.
See https://github.com/getsentry/sentry-docs/pull/11300#discussion_r1753216612